coralproject/cay
# Ask 125

## What does this PR do?

This PR solves issue 125 https://github.com/coralproject/ask/issues/125
## How do I test this PR?
- Create a new form
- Add some field, edit something
- Go to another route without saving
- Should trigger an alert
- Check for expected behaviour (description https://github.com/coralproject/ask/issues/125)

@coralproject/frontend
